app/bundle/bundle.sbt

```diff
@@ -11,8 +16,50 @@ assembly / mainClass := Some("org.bitcoins.bundle.gui.BundleGUI")
 
 assembly / assemblyJarName := s"${name.value}.jar"
 
+//need compatibility with windows versioning scheme which is
+//w.x.y.z
+Windows / version := previousStableVersion.value.get
+
 assembly / assemblyMergeStrategy := {
   case PathList("META-INF", _ @_*)       => MergeStrategy.discard
   case PathList("reference.conf", _ @_*) => MergeStrategy.concat
   case _                                 => MergeStrategy.first
 }
+
+Compile / doc := (target.value / "none")
+// general package information (can be scoped to Windows)
+maintainer := "Chris Stewart <stewart.chris1234@gmail.com>"
+// Will say "Welcome to the <packageSummary> Setup Wizard"
+packageSummary := "Bitcoin-S"
+// Will be used for drop down menu in setup wizard
+packageDescription := "Bitcoin-S"
+
+// wix build information
+wixProductId := java.util.UUID.randomUUID().toString
+wixProductUpgradeId := java.util.UUID.randomUUID().toString
+
+// Adding the wanted wixFeature:
+wixFeatures += WindowsFeature(
+  id = "shortcuts",
+  title = "Shortcuts in start menu",
+  desc = "Add shortcuts for execution and uninstall in start menu",
+  components = Seq(
+    AddShortCuts(Seq("bin/bitcoin-s-bundle.bat"))
+  )
+)
+
+// for windows class paths being too long
+scriptClasspath := Seq("*")
```

Comment:
So is all this stuff needed? My understanding is this all this is extra configuration that wasn't needed because we weren't uninstalling the program correctly.

Reply:
No, the uninstall process made it so I wasn't correctly installing any of the changes I made so I wasn't sure what changes I did worked.